### ISSUE TITLE
Lab extension compatibility glitch

### ISSUE THREAD

**Issue by bosr** [jupyterlab/jupyterlab-github#15]
Hi @ian-r-rose,

What an amazing work ❤️ 

Would you consider updating the version in `package.json` for this extension to work with the current Jupyter labextension 0.12.0+ version? I currently have this error (JPLab 0.29.1) when trying to install the labextension:

```
ValueError:
"@jupyterlab/github@0.1.1" is not compatible with the current JupyterLab
Conflicting Dependencies:
JupyterLab              Extension            Package
>=0.12.0-0 <0.13.0-0    >=0.11.0-0 <0.12.0-0 @jupyterlab/application
>=0.12.4-0 <0.13.0-0    >=0.11.0-0 <0.12.0-0 @jupyterlab/apputils
>=0.12.0-0 <0.13.0-0    >=0.11.0-0 <0.12.0-0 @jupyterlab/coreutils
>=0.12.0-0 <0.13.0-0    >=0.11.0-0 <0.12.0-0 @jupyterlab/docmanager
>=0.12.0-0 <0.13.0-0    >=0.11.0-0 <0.12.0-0 @jupyterlab/filebrowser
>=0.51.0-0 <0.52.0-0    >=0.50.0-0 <0.51.0-0 @jupyterlab/services
```

Cheers

**Comment by ian-r-rose** [comment]
Thanks for the kind words! Yes, I will be updating this tomorrow, or at the latest this weekend (travelling at the moment).

**Comment by ian-r-rose** [comment]
Hi @bosr , I have just published v0.2.0, let me know if you have any troubles.

**Comment by bosr** [comment]
Thanks, it works perfectly.
